src/bci_build/package/appcontainers.py

```diff
@@ -724,3 +724,32 @@ def _get_nginx_kwargs(os_version: OsVersion):
     )
     for tomcat_major, os_version in product(_TOMCAT_VERSIONS, ALL_BASE_OS_VERSIONS)
 ]
+
+APACHE2_CONTAINERS = [
+    ApplicationStackContainer(
+        name="apache2",
+        os_version=os_version,
+        package_name="apache2-image",
+        pretty_name="Apache HTTPD Server",
+        is_latest=os_version in CAN_BE_LATEST_OS_VERSION,
+        version="%%apache2_version%%",
+        version_in_uid=False,
+        replacements_via_service=[
+            Replacement(
+                regex_in_build_description="%%apache2_version%%",
+                package_name="apache2",
+                parse_version="minor",
+            )
+        ],
+        license="Apache-2.0",
+        package_list=["apache2"],
```

Comment:
We offer various MPM modules, should the name at least reflect the MPM that was chosen?

Reply:
That is a good question, but I'm afraid I can't answer that. Someone else needs to clarify the purpose of this image.